### PR TITLE
kdump: remove last usages of PF3 and stop including it

### PR DESCRIPTION
**Pull request by KKoukiou**
Main page on master:
image

Main page on this PR:
image

location dialog on master:
image

location dialog on this PR:
image

mobile location dialog on master:
image

mobile location dialog on this PR:
image

### CODE REVIEW

**Comment by KKoukiou** [comment]
Some failing tests - I will repost.

**Comment by KKoukiou** [comment]
Still some test failures

**Comment by martinpitt** [comment]
[This failure](https://logs.cockpit-project.org/logs/pull-15498-20210311-120011-da67f577-ubuntu-stable/log.html#263) is another fallout from our recent sudo/unexpected messages rework. Do you want to add a fix here, or want me to send a separate PR?

**Comment by KKoukiou** [comment]
> [This failure](https://logs.cockpit-project.org/logs/pull-15498-20210311-120011-da67f577-ubuntu-stable/log.html#263) is another fallout from our recent sudo/unexpected messages rework. Do you want to add a fix here, or want me to send a separate PR?

Let's send a seperate PR, if this is not related to these changes.

**Comment by martinpitt** [comment]
I made some screenshots for comparing:
 * Main page: The circled i got replaced with a circled ?. That's OK and just PF4-ish, I figure?
 * location dialog on desktop has much more spacing  between label and input fields. Again, I suppose standard PF4?
 * location dialog on mobile is better now, as it moves the labels on top of the input fields, leaving more space to them. It also fixes the odd spacing for the checkbox.

So, nothing that I recognize as blocking, just wanted to practice spotting visual differences a bit (hey @garrett :grin: )

**Review by martinpitt — APPROVED** [review]
LGTM, thank you!

File: pkg/kdump/kdump-view.jsx [hunk 1]
```diff
@@ -456,24 +450,28 @@ export class KdumpPage extends React.Component {
...
-                        <span className="fa fa-lg fa-info-circle" />
-                    </button>
+                    <OutlinedQuestionCircleIcon className="popover-ct-kdump" />
                 </Tooltip>
             );
         }
         var serviceWaiting;
         if (this.props.stateChanging)
-            serviceWaiting = <div className="spinner spinner-sm" />;
+            serviceWaiting = <Spinner size="md" />;
```
Comment (marusak) [inline pkg/kdump/kdump-view.jsx:474]: This is (in my option) misaligned:
aligningcoredumps

Both spinner and text are top aligned with switch, which I don't think is right.

**Comment by marusak** [comment]
I just noticed this one misalignment but other than that nice rewrite, thanks!

File: pkg/kdump/kdump-view.jsx (hunk 1 above)
Comment (martinpitt) [reply pkg/kdump/kdump-view.jsx:474]: Well spotted, I didn't notice that yesterday. That misalignment already happens on master though (see my screenshots).

Comment (marusak) [reply pkg/kdump/kdump-view.jsx:474]: > already happens on master

Ah right, haven't checked against master. I'll let @KKoukiou to decide if she wants to fix it as part of this PR or if we are going to do it as followup.

Comment (KKoukiou) [reply pkg/kdump/kdump-view.jsx:474]: @marusak @martinpitt I will post follow up (if I can figure it out) or open an issue (if not).

Comment (KKoukiou) [reply pkg/kdump/kdump-view.jsx:474]: https://github.com/cockpit-project/cockpit/issues/15511

**Pull request merged**